backend/app/main/survey_views.py

```diff
@@ -23,14 +23,26 @@ def sms_signup():
         ],
         "title": "sign-up form"
         }
+
+        name_inserted = False
         for (key, value) in questions_json["fields"].items():
             kind = "email" if key == "Email" else "text"
             new_question = {
             "text": value,
             "kind": kind,
             "airtable_id": key
             }
-            new_json_form["questions"].append(new_question)
+
+            if key == "Name":
+                new_json_form["questions"].insert(0, new_question)
```

Comment:
I wonder if we could remove the flag variable `name_inserted` since **Name** is the most basic information needed and we could insert the **Email** question to the 2nd place if it exists.

Reply:
Oh, I can't really do new_json_form["questions"].insert(1, email_question) since for example:
Let's say the first one retrieved is Age, then the list would be [age_question].

So, if the next one is Email, it would add it after the age_question: [age_question, email_question]

Reply:
I see, how about do `insert(0, new_question)` and `insert(1, new_question)` for Name and Email first and then append the rest questions?